frontend/src/routes/Identities/Users/UserDetail.tsx

```diff
@@ -1,18 +1,146 @@
+  const { data: users, loading } = useQuery(listUsers)
+  const [userActionsOpen, setUserActionsOpen] = useState<boolean>(false)
+
+  const user = useMemo(() => {
+    if (!users || !id) return undefined
+    return users.find((u) => u.metadata.uid === id || u.metadata.name === id)
+  }, [users, id])
+
+  const getUserActions = useCallback(() => {
+    if (!user) return []
+
+    const actions = [
+      <DropdownItem
+        component="button"
+        key="impersonate-user"
+        onClick={() => {
+          // TODO: Implement impersonate logic
+          console.log('Impersonate user:', user.metadata.name)
+        }}
+      >
+        {t('Impersonate user')}
+      </DropdownItem>,
+      <DropdownItem
+        component="button"
+        key="edit-user"
+        onClick={() => {
+          navigate(generatePath(NavigationPath.identitiesUsersYaml, { id: id ?? '' }))
+        }}
+      >
+        {t('Edit user')}
+      </DropdownItem>,
+      <DropdownItem
+        component="button"
+        key="delete-user"
+        onClick={() => {
+          // TODO: Implement delete user logic
+          console.log('Delete user:', user.metadata.name)
+        }}
+        className="pf-m-danger"
+      >
+        {t('Delete user')}
+      </DropdownItem>,
+    ]
+
+    return actions
+  }, [user, navigate, id, t])
+
+  if (loading) {
+    return (
+      <PageSection>
+        <AcmLoadingPage />
+      </PageSection>
+    )
+  }
+
+  if (!user) {
+    return (
+      <AcmPage
+        header={
+          <AcmPageHeader
+            title={t('User not found')}
+            breadcrumb={[
+              { text: t('User Management'), to: NavigationPath.roles },
+              { text: t('Users, groups and service accounts'), to: NavigationPath.identities },
+              { text: id ?? t('User not found') },
+            ]}
+          />
+        }
+      >
+        <div>{t('User with ID {{id}} was not found', { id })}</div>
+      </AcmPage>
+    )
+  }
+
+  const isDetailsActive = location.pathname === generatePath(NavigationPath.identitiesUsersDetails, { id: id ?? '' })
+  const isYamlActive = location.pathname.includes('/yaml')
+  const isRoleAssignmentsActive = location.pathname.includes('/role-assignments')
+  const isGroupsActive = location.pathname.includes('/groups')
 
   return (
-    <AcmPage header={<AcmPageHeader title={t('User Details')} description={`User: ${id}`} />}>
-      <AcmPageContent id="user-details">
-        <PageSection>User detail page for ID: {id}</PageSection>
-      </AcmPageContent>
+    <AcmPage
+      hasDrawer
+      header={
+        <AcmPageHeader
+          title={user?.fullName ?? t('Unknown user')}
+          description={user.metadata.name}
+          breadcrumb={[
+            { text: t('User Management'), to: NavigationPath.roles },
+            { text: t('Users, groups and service accounts'), to: NavigationPath.identities },
+            { text: user.metadata.name ?? t('Unknown user') },
```

Comment:
the breadcrumb is wrong FMPOV
<img width="780" height="180" alt="image" src="https://github.com/user-attachments/assets/334a93fb-c9e3-4a97-8b46-22a9ca9b0972" />

shouldn't it be "User Management -> Identites -> Users -> the user"?